LPS-174758 LPS-200418 portal-search-web: Update Contributor names in FE

gustavolimav opened the pull request.

haydenzheng314 committed:
LPS-197550 Replace non-blocking ServiceProxyFactory usages with ServiceTracker. Can not use Snapshot here because it makes the method _checkBuildDate() get service earlier than the activation of ReleasePublisher and ReleaseManagerImpl. This causes testPublishUpgrade() in ReleasePublisherTest failed.

Commit:
LPS-201309 Expose SXPParameterContributor

Commit:
LPS-201309 Create SXPParameterContributorProvider, SXPParameterContributorProviderImpl, move sxpParameterContributors logic from SXPParameterDataCreator to SXPParameterContributorProviderImpl and use SXPParameterContributorProvider instead

Commit:
LPS-201309 Inline getSXPParameterContributorDefinitions from SXPParameterDataCreator into SXPParameterContributorDefinitionResourceImpl and remove SXPParameterContributorDefinitionProvider implementation from SXPParameterDataCreator

Commit:
LPS-201309 Remove SXPParameterContributorDefinitionProvider interface

Commit:
LPS-201309 Rename SXPParameterDataCreator to SXPParameterDataCreatorUtil, make this a util class, pass sxpParameterContributors as a parameter to create method and fix SXPBlueprintSearchRequestEnhancerImplTest

igor-franca committed:
LPS-201599 Create the Necessary Language Keys to Export and Import Object Folders

igor-franca committed:
LPS-201599 buildlang

igor-franca committed:
LPS-201599 Add Import and Export Options in Dropdown Menu Actions

Commit:
LPS-195586 LPS-202411 search-experiences-rest-impl: Encode elementdefinition and elementInstance to avoid security vulnerability in LXC environment

Commit:
LPS-195586 LPS-202201 search-experiences-web: Add encoding for strings within elementDefinition JSON

Commit:
LPS-201309 Remove unused paramater

Commit:
LPS-195586 LPS-202542 search-experiences-service: Fix OOTB Elements

Commit:
LPS-195586 LPS-202542 search-experiences-service: Rename file to match element name

pedro-oliveira446 committed:
LPS-146782 Remove traiiling zeros on search to avoid problems with some DBs

pedro-oliveira446 committed:
LPS-146782 Adapt search test to new rule

igor-franca committed:
LPS-201599 Make the getObjectFolderActions a Named Function

igor-franca committed:
LPS-202399 Enable the ModalImportObjectDefinition to be called by a react component

Commit:
LPS-195586 LPS-202543 search-experiences-service: Create Upgrade Process to SXPElements and SXPBlueprints

pedro-oliveira446 committed:
LPS-146782 Change to date instead of date time

igor-franca committed:
LPS-201599 Standardize the Interfaces of HTTP Methods in Objectfolder and Objectdefinition

igor-franca committed:
LPS-201599 Remove Default Import Message to Uncategorized Folder

igor-franca committed:
LPS-201599 buildlang

oliv-yu committed:
LPS-195586 LPS-202544 search-experiences-web: Use 'fieldMappings' value for default when applicable

pedro-oliveira446 committed:
LPS-146782 Assert only the date to avoid problem with oracle DB

igor-franca committed:
LPS-201599 Use Usemodal onClose to Avoid Screen Freezing

pedro-oliveira446 committed:
LPS-146782 Strip trailing zeros of big decimal numbers

pedro-oliveira446 committed:
LPS-146782 Adapt tests to new Big Decimal rule

igor-franca committed:
LPS-201598 Connect Export Object Folder With the MVCResourceCommand

pedro-oliveira446 committed:
LPS-146782 Add new pattern to date time fields

oliv-yu committed:
LPS-195586 LPS-202411 search-experiences-rest-impl: SF

pedro-oliveira446 committed:
LPS-146782 Adapt test to the new agregation rule

Commit:
LPS-195925 Remove the version protection logic(it was addd by LPS-163008(6b5de2cbd549c5e181bbd1d7c0725811aea3a02c)). The condition is too strict and it will miss the lower version control. For example, AssetCategory entity definition on 7.1.x. AssetCategoryPersistenceImpl#updateImpl() should include the else block. Actually, it should match with ServiceBuilder function which was added by LPS-125672(e84f7497cf0fc7a5ab27ae418114a3090745b944).

Commit:
LPS-195925 When if condition is true, it means externalReferenceCode column existed(When entity.hasExternalReferenceCode() is true, it will create the externalReferenceCode column) and we should sanitize the column value. Using classPK as var can avoid the var duplicated. For example, the User_ table exists externalReferenceCode column and its primaryKey is userId.

Commit:
LPS-195925 SF, remove unnecessary codes.

pedro-oliveira446 committed:
LPS-146782 Add entity type date in Oracle DB date rule

igor-franca committed:
LPS-201598 Use Modal Import to Render the Object Folder Import

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in add_commerce_catalog.jsp

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in add_commerce_channel.jsp

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in edit_commerce_notification_template.jsp

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in add_commerce_discount.jsp

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in details.jsp

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in commerce_discounts/qualifiers.jsp

pedro-oliveira446 committed:
LPS-203002 Remove trailing zeros of aggregations numbers

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in add_commerce_discount_rule.jsp

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in commerce_price_lists/qualifiers.jsp

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in info.jsp

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in add_commerce_pricing_class.jsp

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in edit_cp_definition_option_rel.jsp

DiegoHu97 committed:
LPS-202406 Remove not used liferay-portlet-url usages

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in general.jsp

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage add_cp_option.js

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage add_cp_option_value.js

DiegoHu97 committed:
LPS-202406 Add missing language key

DiegoHu97 committed:
LPS-202406 buildLang

igor-franca committed:
LPS-203041 Use dbTableName to Add Visual Changes to a Dummy Object Definition

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

gustavolimav commented:
ci:test:sf

gustavolimav commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-21.liferay.com/job/test-portal-source-format/7410/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-21.liferay.com/userContent/jobs/test-portal-source-format/builds/7410/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-21.liferay.com/job/test-portal-source-format/7411/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-21.liferay.com/userContent/jobs/test-portal-source-format/builds/7411/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 7 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/aa3f9210344b7a7df0391562d3ae89f27f373ad7">aa3f9210344b7a7df0391562d3ae89f27f373ad7</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/gustavolimav/liferay-portal/tree/LPS-174758">LPS-174758</a><br/>Branch GIT ID: <a href="https://github.com/gustavolimav/liferay-portal/commit/82b475d192b0fa7d428d7acdf37bb13fe4fc2bec">82b475d192b0fa7d428d7acdf37bb13fe4fc2bec</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-21.liferay.com/job/test-portal-source-format/7410/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-21.liferay.com/userContent/jobs/test-portal-source-format/builds/7410/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-21.liferay.com/job/test-portal-source-format/7411/">test-portal-source-format#7411</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-21.liferay.com/userContent/jobs/test-portal-source-format/builds/7411/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/gustavolimav/liferay-portal/pull/66">gustavolimav#66</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2470279255">[master] ci:test:sf - gustavolimav &gt; gustavolimav - PR#66 - 2023-12-04[08:53:46]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-21.liferay.com/job/publish-testray-report/3598/">publish-testray-report#3598</a><br/></div></div>

Commit:
LPS-201309 Semver

# breaking

## What modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/blueprint/parameter/contributor/SXPParameterContributor.java

Move SXPParameterContributor from search-experiences-service module to search-experiences-api module

## Why

This change is necessary for another module's class to have access to the interface

----

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-17.liferay.com/job/test-portal-source-format/6684/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-17.liferay.com/userContent/jobs/test-portal-source-format/builds/6684/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ee4b66923fceca0dd3fdf63c33093bca3c89fd8f">ee4b66923fceca0dd3fdf63c33093bca3c89fd8f</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/gustavolimav/liferay-portal/tree/LPS-174758">LPS-174758</a><br/>Branch GIT ID: <a href="https://github.com/gustavolimav/liferay-portal/commit/12f7b16974e8117fc464bb7c7341e4e08b1b9751">12f7b16974e8117fc464bb7c7341e4e08b1b9751</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-17.liferay.com/job/test-portal-source-format/6684/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-17.liferay.com/userContent/jobs/test-portal-source-format/builds/6684/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-17.liferay.com/job/test-portal-source-format/6684/">test-portal-source-format#6684</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-17.liferay.com/userContent/jobs/test-portal-source-format/builds/6684/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/gustavolimav/liferay-portal/pull/66">gustavolimav#66</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2470900862">[master] ci:test:sf - gustavolimav &gt; gustavolimav - PR#66 - 2023-12-05[08:22:25]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-17.liferay.com/job/publish-testray-report/3665/">publish-testray-report#3665</a><br/></div></div>

gislayne-vitorino committed:
LPS-203295 (image-impl) Change location of ImageTableRefenceDefinition so it could be located in the image module

gislayne-vitorino committed:
LPS-203295 (image-test) Create image-test module and add test for ImageTableReferenceDefinition

gislayne-vitorino committed:
LPS-203295 (image-impl) Update query parameter to properly retrieve and delete the CTSContent when deleting Image

igor-franca committed:
LPS-203041 Improve ObjectFolderCardHeader Tests

igor-franca committed:
LPS-203041 Improve ObjectFoldersSidebar Test

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for liferay-partner-etc-cron

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for liferay-sample-etc-cron

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for liferay-ticket-etc-cron

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for osb-faro-engine-client

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for osb-spring-boot-client-zendesk

tinatian committed:
LPS-195925 Only sanitize when externalReferenceCode is changed

BarbaraCabrera committed:
LPS-202230 Create MBConfigurationDisplayContext

BarbaraCabrera committed:
LPS-202230 Add necessary methods to the configuration Display Context

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for liferay-sample-etc-spring-boot

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for liferay-jethr0-etc-spring-boot

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for liferay-evp-etc-spring-boot

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for liferay-customer-etc-spring-boot

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for liferay-clarity-etc-spring-boot

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for liferay-ticket-etc-spring-boot

Seiphon committed:
LPS-203589 Upgrade Spring Boot Version for liferay-marketplace-etc-spring-boot

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for client-extension-util-spring-boot

Seiphon committed:
LPS-203589 Upgrade Spring Boot version for gradle-plugins-defaults

BarbaraCabrera committed:
LPS-202230 Add new configuration jsp to show the new view and add this view under feature flag in MBAdminConfigurationAction and remove it from MBAdminPortlet

larissacribeiro committed:
LPS-202418 Create ObjectFieldReadOnlyExceptionMapper

larissacribeiro committed:
LPS-202418 Render read only in relationship field as disabled for root descendant node

ericyanLr committed:
LPS-203443 Patch to fallback to original logic when IntrospectionException is encountered

This allows our own code to continue using our optimized logic.
As for foreign code that may encounter errors when using a more
complex set of getter/setter methods, we can fallback to using
the original logic.

ericyanLr committed:
LPS-203443 Bump version

ericyanLr committed:
LPS-203443 Apply

ericyanLr committed:
LPS-203443 ant build-lib-versions

BarbaraCabrera committed:
LPS-202230 Add the necessary elements for the new view

ccorreagg committed:
LPS-203406 REST Builder - Set and get the parameter using internally a Supplier, then, it will be called only when it is needed. With that, we will avoid settings always the information if the attributes are not used

Co-authored-by: Alejandro Tardín <alejandro.tardin@liferay.com>

ccorreagg committed:
LPS-203406 Avoid parsing the JSON and use the getter method instead, as we don't need to parse the whole JSON. That would cause invoking all the getter methods even though we only need the information from the id

ccorreagg committed:
LPS-203406 buildREST

ccorreagg committed:
LPS-203406 Use setters with the Suppliers to avoid filling the whole DTO with all the information. That will be done in case the user has requested the fields through fields/restrictFields (by default, all the fields will be returned in the REST APIs)

julschong committed:
LPS-203925 Merge portal-impl LayoutSetModelListener into layout-impl LayoutSetModelListener

julschong committed:
LPS-203925 Change clearCache method to private

julschong committed:
LPS-203925 Add null checking to onBeforeRemove method to avoid throwing NPE

julschong committed:
LPS-203925 Remove LayoutSetModelListener from model-listener-spring

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in add_cp_definition.jsp

DiegoHu97 committed:
LPS-202406 Replace liferay-portlet-url usage in duplicate_cp_definition.jsp

ccorreagg committed:
LPS-203406 Add suppressions, as we will ask for changes in the SF rule. Now, we want to use the setter with UnsafeSupplier to avoid filling the whole DTO with the fields that the user don't need/want to be returned

ccorreagg committed:
LPS-203406 Modify tests

Commit:
LPS-195925 This entity.hasExternalReferenceCode() is unnecessary in this if check. When entity.hasExternalReferenceCode() returns "true", it will create "externalReferenceCode" column automatically. This means that entity.hasEntityColumn("externalReferenceCode") is always true when entity.hasExternalReferenceCode()is true. entity.hasEntityColumn("externalReferenceCode") condition alone is enough.

Note:
When entity.hasEntityColumn("externalReferenceCode") is true, entity.hasExternalReferenceCode() can be either true or false.
When entity.hasEntityColumn("externalReferenceCode") is false, entity.hasExternalReferenceCode() is false.
For more information, please refer to the commit LPS-125672 (e84f7497cf0fc7a5ab27ae418114a3090745b944) changes in the ServiceBuilder class.

victorg1991 committed:
LPS-204013 Create display context

victorg1991 committed:
LPS-204013 Update jsp to extract java code and to use clay components

sandrodw3 committed:
LPS-199879 Remove metal-uri from client.js

ccorreagg committed:
LPS-203406 Use the getter method if it is available, otherwise use the field. With that, we will avoid calculating the fields in case the export does not contain those fields

victorg1991 committed:
LPS-204013 Add radio button to select required in asset libraries

victorg1991 committed:
LPS-204013 Adjust auto fields to make it work properly with radio buttons

victorg1991 committed:
LPS-204013 buildLang

ruben-pulido committed:
LPS-204048 Update feature flag title and description for LPS-196768

Co-authored-by: Eric Moritsuka <ericthmoritsuka@gmail.com>

victorg1991 committed:
LPS-204013 Allow passing depot required

victorg1991 committed:
LPS-204013 We need to get the connected asset libraries too

victorg1991 committed:
LPS-204013 Save required and depot required when saving the vocabulary

ccorreagg committed:
LPS-203406 REST Builder - Update toString method generation too in order to use the getters

carolmariaabb committed:
LPS-155021 Also include nested fields

carolmariaabb committed:
LPS-155021 Make sure the predefined value is defined before parsing

igor-franca committed:
LPS-202923 Make Modal Import Object Definition More Generic and use in List Type Definition

victorg1991 committed:
LPS-204013 Remove as much AUI as possible

Since we cannot remove all because auto_fields rely on it

victorg1991 committed:
LPS-204013 baseline

sandrodw3 committed:
LPS-199879 Remove metal-uri from router.ts

sandrodw3 committed:
LPS-199879 Remove param with no value

sandrodw3 committed:
LPS-199879 Update snapshots

sandrodw3 committed:
LPS-199879 Remove dependency

4lejandrito committed:
LPS-204334 Using a different instance each time

Otherwise the headers are not populated correctly here:

https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/jaxrs/exception/mapper/BaseExceptionMapper.java#L65-L66

pedro-oliveira446 committed:
LPS-203001 Avoid null pointer

victorg1991 committed:
LPS-204013 Rename

Commit:
LPS-195586 LPS-202411 search-experiences-rest-impl: Add null validation

Commit:
LPS-195586 LPS-202411 portal-web: Fix poshi test

mbowerman committed:
LPS-204350 Ensure that the feature to properly redirect when there is an
invalid language ID in the URL works for all pages, not just top-level
pages

larissacribeiro committed:
LPS-202418 Throw exception when trying to change read only value for root descendant node

igor-franca committed:
LPS-203041 Update Test Locator

michaelhashimoto committed:
LRCI-3937 Add plugins jobs to the liferay database

michaelhashimoto committed:
LRCI-3937 Add plugins job entity classes

michaelhashimoto committed:
LRCI-3937 Add plugins job definition classes

michaelhashimoto committed:
LRCI-3937 Rename QA Websites Query to Poshi Query for job parameters

michaelhashimoto committed:
LRCI-3937 Add Plugins Extra Apps job & build parameters

Commit:
LPS-204363 Application Menu has menuitems, not links, and "Objects" should be an exact match.

michaelhashimoto committed:
LRCI-3937 Add Plugins Marketplace Apps job & build parameters

michaelhashimoto committed:
LRCI-3937 Add Plugins Upstream & Release job & build parameters

michaelhashimoto committed:
LRCI-3937 Add Plugins Pull Request job & build parameters

shuyangzhou committed:
LPS-204437 Optimize user's site groups query

shuyangzhou committed:
LPS-204437 Avoid getting unnecessary attributes from Contact

shuyangzhou committed:
LPS-204437 Avoid "duplicated" service calls

shuyangzhou committed:
LPS-204437 Avoid getting unnecessary attributes from exporting baseModel

Commit:
LPS-204374 Test fix for ExportTaskResourceTest.testPostExportTask

Commit:
LPS-204375 Test fix to update path if objects navigation

Commit:
LPS-195925 SF.

Commit:
LPS-195925 Auto generated

4lejandrito committed:
LPS-203406 Not needed

sergiojimcos committed:
LPS-204388 Remove previous job scheduler using its old DisdatchTaskClusterMode

ccorreagg committed:
LPS-203406 Avoid analyzing the Suppliers as it is not a DTO field, it is internal

ccorreagg committed:
LPS-203406 Include extra conditions to avoid possible usage of methods with the expected name but unexpected arguments or return type

ccorreagg committed:
LPS-203406 Fix method name for enums created with REST Builder

ccorreagg committed:
LPS-203406 Add test

BarbaraCabrera committed:
LPS-202230 Add the EmailFrom validator under the FF

BarbaraCabrera committed:
LPS-202230 Add the additionalProps attribute to the clay select to be able to use it with the propsTransformer attribute

BarbaraCabrera committed:
LPS-202230 Move updateRanks, updatePriorities and save configuration logic to propsTransformer

ambrinchaudhary committed:
LPS-200534 Remove buttons after success response

ambrinchaudhary committed:
LPS-200534 Style buttons

titokenzo committed:
LPS-204047 Fix modal form export entries layout

BarbaraCabrera committed:
LPS-202230 Fix save and cancel buttons position

p2kmgcl committed:
LPS-203513 Disable input when configuring non-default Display Pages

sergiojimcos committed:
LPS-204388 Add test case

sergiojimcos committed:
LPS-204388 Use methods

ealonso committed:
LPS-204366 Don't check it if we are deleting a group

marcosapmf committed:
LRAC-15298 rename functions with the same to prevent one from being overwritten by the other

igor-franca committed:
LPS-203041 Update Language Keys in ModalImport

4lejandrito committed:
LPS-204388 Not needed, this does not happen

Commit:
LPS-202509 Dev Workspace: Add a substitution variable containing the build time that can be used in CX URLs

4lejandrito committed:
LPS-204388 Declaring as late as possible

igor-franca committed:
LPS-203041 Update Language Keys

victorhcunha committed:
LPS-204427 Fix assertIsDisplayedOnControlPanel macro

victorg1991 committed:
LPS-202962 Bring back customization option for smaller screens

peerkar committed:
LPS-196453 portal-search-opensearch2-impl: Engine adapter API / Document

peerkar committed:
LPS-196453 portal-search-opensearch2-impl: Engine adapter API / Search

victorhcunha committed:
LPS-204427 Fix tests that assert the object definition in the applications menu

peerkar committed:
LPS-196453 portal-search-opensearch2-impl: Engine adapter impl

shuyangzhou committed:
LPS-204437 Expose bulk fetch method

victorhcunha committed:
LPS-204427 Remove unnecessary macro

interaminense committed:
LPS-199879 Auto SF

pedro-oliveira446 committed:
LPS-203001 Semver

samuelleolopes committed:
ISSD-2129 Created Mega Menu item for Users to Escalate a Ticket

brianchandotcom committed:
LPS-204437 SF

victorhcunha committed:
LPS-204427 Fix CanAccessCustomObjectWithAccessPermission test

brianchandotcom committed:
LPS-204366 Sort

victorhcunha committed:
LPS-204427 Remove unnecessary macro

marcosapmf committed:
LRAC-15299 check if target content matches to download language value

brianchandotcom committed:
LRCI-3937 Wordsmith

brianchandotcom committed:
LPS-204013 Wordsmith

brianchandotcom committed:
LPS-204013 Regen

brianchandotcom committed:
LPS-204013 Rename

shuyangzhou committed:
LPS-204437 Regenerate

brianchandotcom committed:
LPS-196453 SF

shuyangzhou committed:
LPS-204437 Bulk fetch contact and pre-install them into users to avoid individual contact db fetching.

brianchandotcom committed:
LPS-203981 prep next

interaminense committed:
LPS-199879 Update yarn.lock

brianchandotcom committed:
LPS-203981 LPS-203589 prep next

kyle-miho committed:
LRQA-83508 Add information that no test failures are in a classified category instead of leaving it empty to prevent confusion

igor-franca committed:
LPS-203041 buildlang

kyle-miho committed:
LRQA-83508 Add extra error messages that will categorize a testcase failure as DID_NOT_RUN

oliv-yu committed:
LPS-195586 LPS-202544 search-experiences-web: Extend timeout for js tests

xbrianlee committed:
LPS-204354 Update for LPS-203725 (additional commit)

Commit:
LPS-202680 Modify locator to make it reusable

interaminense committed:
LRAC-14397 Follow up - fixing unit tests

Commit:
LPS-202680 Add locator for highlighted structures subtitle

Commit:
LPS-202680 Add macro to select highlighted structures

Commit:
LPS-202680 Add case to manage web contents by structures

peerkar committed:
LPS-196453 portal-search-opensearch2-impl: The rest of the classes

ruben-pulido committed:
LPS-204419 If feature flag is enabled, display SearchOptions component only if search has been performed (regardless of whether filtering has been applied or not)

ruben-pulido committed:
LPS-204419 Display SearchIn button only if search has been performed (regardless of feature flag value)

JamesAGarcia committed:
LPS-204048 updates ff title and description

ruben-pulido committed:
LPS-204419 Display Search Location button only if search has been performed

ruben-pulido committed:
LPS-204419 Consider only keywords to determine if search has been performed (regardless of feature flag value)

ruben-pulido committed:
LPS-204419 Add missing return in lambda

ruben-pulido committed:
LPS-204419 If search has not been performed return null for searchLocationOptions

shuyangzhou committed:
LPS-204437 Setup an enclosing tx boundary to avoid unnecessary sub-aop boundaries overhead.

JamesAGarcia committed:
LPS-204048 adds ff type

Commit:
LPS-202680 Add case to view web contents in highlighted structure sheet

ruben-pulido committed:
LPS-204419 Don't hide folder breadcrumb when mine or recent filters have been applied

Commit:
LPS-202680 Add case to search web contents in highlighted structure sheet

ruben-pulido committed:
LPS-204419 Ensure that the tab parameter is replaced (and not just added).

Otherwise, switching between web-content, versions and comments when feature flag is disabled was not working correctly.

Commit:
LPS-202680 Add case to search web contents in Content Library sheet

Commit:
LPS-202680 Add case to manage web contents by structures in asset library

ruben-pulido committed:
LPS-204419 Mark "all" as active is navigation is home

ruben-pulido committed:
LPS-204419 Set navigation for mine and recent

Commit:
LPS-202680 Add case to view available types in Content Library sheet

Commit:
LPS-202680 Add locators for info panel

Commit:
LPS-202680 Add macro to check list checkbox

Commit:
LPS-202680 Add case to view info panel of web contents in highlighted structure sheet

qz1010 committed:
LPS-203869 Auto SF

Commit:
LPS-202680 Add case to select structures from global and asset library as highlighted structrues

Commit:
LPS-202680 Add case to view accessibility of highlighted structures

brianchandotcom committed:
LPS-204437 SF

brianchandotcom committed:
LPS-203443 prep next

brianchandotcom committed:
LPS-202418 Wordsmith

JamesAGarcia committed:
LPS-204048 removes 'also' from description

brianchandotcom committed:
LPS-202509 prep next

brianchandotcom committed:
LPS-202509 Use latest

brianchandotcom committed:
LPS-202509 prep next

brianchandotcom committed:
LPS-202509 prep next

brianchandotcom committed:
LRQA-83508 prep next

brianchandotcom committed:
LPS-201599 Wordsmith

brianchandotcom committed:
LPS-201599 Regen

brianchandotcom committed:
LPS-195586 Fix bad naming

brianchandotcom committed:
LPS-202230 SF

brianchandotcom committed:
LPS-202230 Regen

brianchandotcom committed:
LPS-203406 SF

brianchandotcom committed:
LPS-203406 Rename

brianchandotcom committed:
LPS-203406 prep next

brianchandotcom committed:
LPS-203406 prep next

brianchandotcom committed:
LPS-195925 prep next

Commit:
LPS-174758 LPS-204487 portal-search: Add more parameters to webcacheItem because it must have unique name
